src/components/Menu/Menu.stories.tsx

```diff
@@ -44,15 +45,72 @@ export const Default: StoryObj<MenuProps> = {
           Not Possible (disabled)
         </Menu.Item>
       </Menu.Items>
-    </Menu>
-  ),
+    ),
+  },
 };
 
 export const WithLongButtonText: StoryObj<MenuProps> = {
+  args: {
+    buttonText: 'Long Trigger Button Text to Demonstrate Popover Matching',
+    children: (
+      <Menu.Items data-testid="menu-content">
+        <Menu.Item
+          href="https://headlessui.com/react/menu#menu-button"
+          icon="link"
+        >
+          Headless UI Docs
+        </Menu.Item>
+        <Menu.Item
+          href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
+          icon="link"
+        >
+          MDN: Menu
+        </Menu.Item>
+        {/* eslint-disable-next-line no-alert */}
+        <Menu.Item onClick={() => alert('Item clicked')}>
+          Trigger Action
+        </Menu.Item>
+        <Menu.Item disabled href="https://example.org/" icon="warning">
+          Not Possible (disabled)
+        </Menu.Item>
+      </Menu.Items>
+    ),
+  },
+};
+
+export const WithShortButtonText: StoryObj<MenuProps> = {
+  args: {
+    buttonText: 'Menu',
+    children: (
+      <Menu.Items data-testid="menu-content">
+        <Menu.Item
+          href="https://headlessui.com/react/menu#menu-button"
+          icon="link"
+        >
+          Headless UI Docs
+        </Menu.Item>
+        <Menu.Item
+          href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/menu"
+          icon="link"
+        >
+          MDN: Menu
+        </Menu.Item>
+        {/* eslint-disable-next-line no-alert */}
+        <Menu.Item onClick={() => alert('Item clicked')}>
```

Comment:
Future idea: we could experiment with using argTypes to generate Storybook "actions" for things like this: https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation.

Would tie into Storybook a little more. But wouldn't be as obvious as an alert. So not sure which is better, or if it really matters.

Reply:
That makes sense to me, can add to this pr as well

Reply:
Finding it's difficult to add argType for this since it's an argument for the subcomponent `Menu.Item` and the Args for these stories are for `Menu`